packages/open-next/src/core/requestHandler.ts

```diff
@@ -65,6 +66,34 @@ export async function openNextHandler(
         delete headers[rawKey];
       }
 
+      if (
+        "isExternalRewrite" in preprocessResult &&
+        preprocessResult.isExternalRewrite
+      ) {
+        try {
+          preprocessResult = await globalThis.proxyExternalRequest.proxy(
+            preprocessResult.internalEvent,
+          );
+        } catch (e) {
+          error("External request failed.", e);
+          preprocessResult = {
+            internalEvent: {
+              type: "core",
+              rawPath: "/500",
+              method: "GET",
+              headers: {},
+              url: "/500",
+              query: {},
+              cookies: {},
+              remoteAddress: "",
+            },
+            isExternalRewrite: false,
```

Comment:
```suggestion
            isExternalRewrite: true,
```

Reply:
Same as in `middleware.ts` this rewrite is internal (even though initially it was attempting an external rewrite)

Reply:
@conico974 Thanks for the feedback. What about adding a comment in both locations?